Packs/FeedTalos/Integrations/FeedTalos/FeedTalos.yml

```diff
@@ -82,6 +82,16 @@ configuration:
   name: proxy
   type: 8
   required: false
+- defaultvalue: 'true'
+  display: Fetch indicators
+  name: feed
+  type: 8
+  required: false
+- defaultvalue: 'true'
+  display: Fetch indicators
+  name: feed
+  type: 8
+  required: false
```

Comment:
Looks like it happened again, please fix it like you did last time.
<img width="545" alt="Screen Shot 2020-09-16 at 10 24 14" src="https://user-images.githubusercontent.com/54398957/93305269-f0b35c00-f806-11ea-9bd3-06fa84597041.png">